### ISSUE TITLE
Error: Logged in Elsewhere

### ISSUE THREAD

**Issue by FearOfThere** [dumbasPL/csgo-checker#34]
No matter if reinstall or for new accounts. All added accounts have the status "Logged in Elsewhere".

**Comment by KillaBoi** [comment]
+1, also facing this issue as well

**Comment by dumbasPL** [comment]
Yeah, I know this is an issue. I'll try to add cs2 support at some point...
